packages/driver/cypress/integration/e2e/multi-domain/logging.spec.ts

```diff
@@ -0,0 +1,146 @@
+describe('cy.origin logging', () => {
+  const { _ } = Cypress
```

Comment:
Any reason to elect the `lodash` destructure  on big cypress here over importing `lodash`?

Reply:
It's already included in Cypress, so importing it would just add to the spec file size and there would be duplicate lodashes, one in the Cypress bundle and one in the spec bundle. We use this pattern a lot in driver specs. Looks like [this spec](https://github.com/cypress-io/cypress/blob/476f2ad12b11fbf46083f8af510044335f572108/packages/driver/cypress/integration/cypress/command_queue_spec.ts#L1) is the only one where we actually import lodash, but we should probably change that.